### ISSUE TITLE
Kurtosis: Parallelize Consensus node startup

### ISSUE THREAD

**Issue by corduroybera** [berachain/beacon-kit#677]
[Low priority: The returns may not be worth the effort]

Consensus nodes are currently started up serially.

We should be able to start them up in parallel, with the one caveat of setting peers.

Setting peers must be done in the config file and the node has to be restarted.

This means after creating the services in parallel we'll need to add exec commands to edit their peers and start the node.
